app/src/main/java/org/mozilla/fenix/ext/String.kt

```diff
@@ -21,13 +21,13 @@ fun String.replace(pairs: Map<String, String>): String {
 }
 
 /**
- * Try to parse and get host part if this [String] is valid URL.
- * Returns **null** otherwise.
+ * Tries to parse and get host part if this [String] is valid URL.
+ * Otherwise returns the string.
  */
-fun String?.getHostFromUrl(): String? = try {
+fun String.tryGetHostFromUrl(): String = try {
```

Comment:
@NotWoods This looks like a candidate for uplifting to A-C: https://github.com/orgs/mozilla-mobile/projects/38. Let's file an issue?

Reply:
mozilla-mobile/android-components#4472